index.js

```diff
@@ -11,11 +11,10 @@ var string2compact = require('string2compact')
 var bencode = require('bencode')
 var inherits = require('inherits')
 
-var PEX_INTERVAL  = 65000 // just over one minute
+var PEX_INTERVAL = 65000 // just over one minute
```

Comment:
Sorry for the random style changes, but I'm in a habit of writing code that's valid in `standard` now. Lmk if that's not cool.